skins/oasis/modules/ContentDisplayController.class.php

```diff
@@ -1,23 +1,23 @@
 <?php
+
 /**
  * Helper Functions: adds picture attribution info and replaces section edit links with pencil icon and link
  * Fixme: this is not really a proper module
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this "FIXME" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1134)